ci/deploy_app.yaml

```diff
@@ -6,6 +6,7 @@ image_resource:
     tag: slim
 inputs:
   - name: eq-questionnaire-runner
+  - name: image_tag
```

Comment:
As per https://github.com/ONSdigital/eq-pipelines/pull/252#pullrequestreview-720278405, you should make this input optional and fetch from file if it exists, if not fetch from env vars. If neither exist, then it should fail the job.

Reply:
Made optional and tested both scenarios [here for test env](https://concourse.gcp.onsdigital.uk/teams/eq/pipelines/pete-test-env-on-release/jobs/deploy-questionnaire-runner/builds/98) and [here for staging](https://concourse.gcp.onsdigital.uk/teams/eq/pipelines/pete-test-env-on-release/jobs/deploy-questionnaire-runner/builds/97). Thanks!